src/unity/python/turicreate/toolkits/_mps_utils.py

```diff
@@ -150,6 +143,19 @@ def _prepare_network_parameters(arg_dict):
         arr[i] = _ctypes.c_void_p(items[i][1].ctypes.data)
     return items, name, arr, sz
 
+def _prepare_graph_network_parameters(arg_dict):
+    items = []
+    for name, arr in arg_dict.items():
+        if not isinstance(arr, _np.ndarray):
+            arr = _np.array(arr, dtype=_np.float32)
```

Comment:
I think `np.asarray` is more common for this use (this is what numpy uses itself). For instance, if you put in a `np.matrix`, the `asarray` will do a very cheap translation without copying the underlying memory while `np.array` will iterate the matrix and copy manually.

Reply:
Yeah, looks like `np.asarray` makes sense here. The usage of `np.array` was copied from the existing `_prepare_network_parameters` function, so I can update that usage, too.